src/main/java/graphql/schema/idl/TypeDefinitionRegistry.java

```diff
@@ -28,6 +34,11 @@
 
     private final Map<String, ScalarTypeDefinition> scalarTypes = new LinkedHashMap<>();
     private final Map<String, List<ObjectTypeExtensionDefinition>> typeExtensions = new LinkedHashMap<>();
```

Comment:
lets rename this field to match the others

Reply:
Ok - so that is in the other PR to be honest #935 

https://github.com/graphql-java/graphql-java/pull/935/files#diff-f7a2f15780b23ce3535e874aefd70ec2L30

I need #935 in place first and then this schema gen extensions.  I extended the 2nd branch from the former branch to allow progression.  But if your happy with this on this branch - you get 2 for one

Reply:
sure ... lets merge it